content/ecosystem/servers/servers.toml

```diff
@@ -1,6 +1,6 @@
 [[servers]]
 name = "Synapse"
-description = "Synapse is a Matrix homeserver written in Python/Twisted. "
+description = "Synapse is a Matrix homeserver written in Python/Twisted."
 author = "Element"
 maturity = "Stable"
 language = "Python"
```

Comment:
I just saw this - do you not consider synapse to also be written in rust? Sure, it doesn't have the rust federation worker, but...

Reply:
There is a little rust, but it is minimal i.e. 1.1% Rust vs 0.4% HTML. 

We have no short/medium term intention to increase, so given this is a very high-level overview, it seems misleading to say it is written in Rust. The github repo does reference rust, though in that context it probably _is_ reasonable to give the extra detail.